Libs/Optimize/ParticleSystem/itkParticleImageDomainWithCurvature.h

```diff
@@ -80,21 +84,25 @@ class ParticleImageDomainWithCurvature : public ParticleImageDomainWithHessians<
     // Release the memory in the parent hessian images.
     //this->DeletePartialDerivativeImages();
     
+    m_CurvatureInterpolator = ScalarInterpolatorType::New();
     m_CurvatureInterpolator->SetInputImage(m_CurvatureImage);
   } // end setimage
   
   double GetCurvature(const PointType &pos) const
   {
-    return m_CurvatureInterpolator->Evaluate(pos);
+    if (m_CurvatureInterpolator) {
```

Comment:
Is this using `m_CurvatureInterpolator` as a proxy for whether or not we are in `minimal` mode?

Reply:
Sort of.  I'm skipping the generation of the gradient/hessian images for the fixed domains since the data will never be used.  Unfortunately, there is a still a callback during AddPosition that ends up asking for the curvature.  Without this check, it will crash.  The curvature doesn't matter for the fixed domains.

The initialization structure needs to be reorganized for sure, this is sort of a temporary kludge to skip some parts for now.